src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -85,6 +85,8 @@ protected static MessageDigest getLabelDigestFunction() {
 
     private String id;
 
+    private boolean unwrapped;
```

Comment:
```suggestion
    private transient boolean unwrapped;
```

since IIUC this is only an optimization for in-memory work?